packages/mui-base/src/className/index.ts

```diff
@@ -1,2 +1,2 @@
 // eslint-disable-next-line import/prefer-default-export
-export { default as unstable_ClassNameGenerator } from './ClassNameGenerator';
+export { unstable_ClassNameGenerator } from '@mui/private-classnames/className';
```

Comment:
```suggestion
export { unstable_ClassNameGenerator } from '@mui/private-classnames';
```

What do you think about using methods from the package index? The package is pretty small and only deal with classname.

Reply:
Yeah makes sense, it doesn't have to be isolated anymore 👍